src/lib.rs

```diff
@@ -61,8 +63,8 @@ impl InlinerBuilder {
         self
     }
 
-    /// Configures whether unexpanded modules (due to a missing file) will lead
-    /// to an `Err` return value or not.
+    /// Configures whether unexpanded modules (due to missing files or invalid Rust sourcd code)
+    /// will lead to an `Err` return value or not.
     ///
     /// Default: `false`.
     pub fn error_not_found(&mut self, error_not_found: bool) -> &mut Self {
```

Comment:
Should this be changed to `report_errors` or something? It's not just about cases where the file isn't found.